src/index.ts

```diff
@@ -152,9 +152,10 @@ export function apply(ctx: Context, config: Config) {
         ucPreset: 2,
         qualityToggle: false,
       }
+      
+      let image: ImageData
```

Comment:
This can be grouped with above code for better SoC:

```ts
let imgUrl: string, image: ImageData
```

Reply:
Well, I always declare them in separated lines even in the same scope. 🤣 
But-uh, I am willing to do the changes as you requested. 
Tout le plaisir est pour moi, monsieur ! 🎉

Reply:
You can also write them on two lines if you like:

```ts
let imgUrl: string
let image: ImageData
```

Reply:
Ça devrait aller maintenant. C'est pas grave.